tests/scr/testSysbox

```diff
@@ -94,6 +93,8 @@ function run_all_tests() {
 
 	printf "\nSysbox health checking ...\n"
 	bats --tap tests/health/sysbox-health.bats
+	printf "\nExecuting life-cycle tests ... \n"
```

Comment:
Why is this change required? And if required, why moving these tests here change anything?

Reply:
So sysbox while it was starting from the testContainerInit file was not respecting the filesystem type. That means that idmapped_mounts was trying to start up on an xfs filesystem which is not supported (yet). In that new [Start sysbox](https://github.com/nestybox/sysbox/pull/584/files/6b3a3328272465ea84e4064d30b1ad96d80ab5f5..98c8d7253ad3fdc7a779ae12652fe1e3888e0540#diff-ca142053720be2d2f7e2b6e884888c9637eb18a7e11a41a26449e4b36160ec91R292) function we do get the output of the host filesystem. If it is xfs for example sysbox is starting without idmapped_mounts `sysbox --disable-idmapped-mount`.

These changes were running as expected but the problem that we had to face with the `testSysbox` and the `run_all_tests` function is that the first tests restarted the sysbox which is reverting the previous checks. These tests were not "respecting" the flag that we set earlier and after the life-cycle tests were done it was like it was running `sysbox` with idmapped_mounts enabled once again. 

So while we discussed that with @ctalledo he proposed to move these tests last

Reply:
@roussidis, thanks for the explanation, understood, I missed this conversation with @ctalledo while being on vacation.

Thanks again for your contribution!

Reply:
My pleasure to help !!